nemo/core/optim/lr_scheduler.py

```diff
@@ -224,6 +224,24 @@ def get_lr(self):
 
         return self._get_lr(step)
 
+# REF: https://arxiv.org/html/2408.11029, with modification on enforcing min_lr
+class WarmupHoldAnnealOneMinusSquareRoot(WarmupHoldPolicy):
```

Comment:
Would be great if a doc-strings could be added, like the other classes in this file. In the doc-string the paper could be referred as well.